utils/dvmhost/Makefile

```diff
@@ -0,0 +1,53 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=dvmhost
+PKG_VERSION:=2025.09.03
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_VERSION:=e9abdf63aa656465438d59f009a83faf5ba69f1f	# R04J32 with hotfixes
+PKG_SOURCE_URL:=https://github.com/dvmproject/dvmhost
+PKG_MIRROR_HASH:=a0346fa9b3d01349f1fced0af17be01a9346bd416aa19fe575ad767538cd11b8
+PKG_BUILD_PARALLEL:=1
+
+PKG_MAINTAINER:=Natalie Moore <natalie@natalielucy.dev>
+PKG_LICENSE:=GPL-2.0-only
+PKG_LICENSE_FILE=LICENSE
+
+# disable TUI to save space; pending upstream updates for websocket
+CMAKE_OPTIONS+= -DENABLE_TUI_SUPPORT=0 -DDISABLE_WEBSOCKETS=1
```

Comment:
If you do want to use websockets, then you can backport your commit, which was merged https://github.com/DVMProject/dvmhost/pull/108/files

Reply:
My plan is to keep no websockets for a "stripped-down" version meant for smaller targets, and a full version with the other utilities and TUI and websockets support later.